cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -144,7 +144,69 @@ module postgreServer 'br/public:avm/res/db-for-postgre-sql/flexible-server:0.1.4
   }
 }
 {{- end}}
-{{- if (or (and .DbMySql .DbMySql.AuthUsingManagedIdentity) (and .DbPostgres .DbPostgres.AuthUsingManagedIdentity))}}
+{{- if .DbMySql}}
+
+var mysqlDatabaseName = '{{ .DbMySql.DatabaseName }}'
+var mysqlDatabaseUser = 'mysqladmin'
+{{- if (and .DbMySql .DbMySql.AuthUsingManagedIdentity) }}
+module mysqlIdentity 'br/public:avm/res/managed-identity/user-assigned-identity:0.2.1' = {
```

Comment:
The identity is only used for MySql. 
Refs:
1. https://learn.microsoft.com/en-us/cli/azure/containerapp/connection/create?view=azure-cli-latest#az-containerapp-connection-create-postgres-flexible(serviceconnector-passwordless)
2. https://learn.microsoft.com/en-us/cli/azure/containerapp/connection/create?view=azure-cli-latest#az-containerapp-connection-create-mysql-flexible(serviceconnector-passwordless)
> ![image](https://github.com/user-attachments/assets/9aaa0514-e21e-4ff7-911a-5f4d51fde7eb)
> ![image](https://github.com/user-attachments/assets/8711c0c6-a412-4275-b4e9-9ca4baeac91e)